### PR TITLE
Add note on OIDC for Auth0

### PR DESCRIPTION
**Pull request by stevenGravy**
Note on turning off OIDC Confformant settings that prevents auth0 connectors from populating claims.  Otherwise a warning will show like "Error validating callback: unable to map claims to role for connector:" in the log.

### CODE REVIEW

File: docs/4.2/oidc.md [hunk 1]
```diff
@@ -37,6 +37,9 @@ documented on the identity providers website. Here are a few links:
    * [Google Identity Platform](https://developers.google.com/identity/protocols/OpenIDConnect)
    * [Keycloak Client Registration](https://www.keycloak.org/docs/latest/securing_apps/index.html#_client_registration)
 
+!!! note For Auth0 the OIDC Conformant setting should be off in Advanced Setting -> OAuth or Claims will not populate properly
```
Comment (webvictim) [inline docs/4.2/oidc.md:40]: For Auth0, the "OIDC Conformant" setting should be off in Advanced Settings -> OAuth or claims will not populate properly (reads a bit better)

Comment (stevenGravy) [reply docs/4.2/oidc.md]: modified

**Review by webvictim — APPROVED** [review]

File: docs/4.2/oidc.md [hunk 2]
```diff
@@ -37,6 +37,11 @@ documented on the identity providers website. Here are a few links:
    * [Google Identity Platform](https://developers.google.com/identity/protocols/OpenIDConnect)
    * [Keycloak Client Registration](https://www.keycloak.org/docs/latest/securing_apps/index.html#_client_registration)
 
+!!! note 
+    
+    For Auth0, the "OIDC Conformant" setting should be off in Advanced Settings -> OAuth or claims will not populate properly
```
Comment (webvictim) [inline docs/4.2/oidc.md:42]: We should add a full stop to the end of the sentence as well.

**Comment by stevenGravy** [comment]
@gravitational-jenkins  retest this please

**Review by benarent — APPROVED** [review]

**Pull request merged**